lib/expression/eval.ex

```diff
@@ -84,9 +85,10 @@ defmodule Expression.Eval do
     ast
     |> Enum.reduce([], &[eval!(&1, context, mod) | &2])
     |> Enum.reverse()
+    |> Enum.map(&not_founds_as_nil/1)
```

Comment:
otherwise the `[not_found: ["foo", "bar"]]` is supplied as a truthy argument to functions like `@IF()` and those would then fail